root/release/edit/information.tt

```diff
@@ -314,110 +314,253 @@
     </p>
   </div>
 
-  <div class="bubble" data-bind="bubble: $root.labelBubble">
-    <!-- ko with: target() && target().label() -->
-      <!-- ko if: $data.gid -->
-        <p data-bind="html: $data.selectionMessage()"></p>
-      <!-- /ko -->
+  <div class="bubble" data-bind="bubble: $root.statusBubble">
+    <!-- ko if: target() && target().statusID() == 4 -->
+      <p>
+        [% l('This release status should be used for unofficial translations
+              and transliterations of tracklists and release titles, and does
+              not denote a separate real release. It should not be used to
+              denote bootlegs, mixtape/street albums, demos, or digital
+              albums. Be sure to link to the corresponding real release with
+              the {url|transl(iter)ation relationship}.',
+             { url => c.uri_for('/relationship/fc399d47-23a7-4c28-bfcf-0607a562b644') }) %]
+      </p>
     <!-- /ko -->
-    <!-- ko if: target() && catNoLooksLikeASIN(target().catalogNumber()) -->
+    <!-- ko ifnot: target() && target().statusID() == 4 -->
       <p>
-        [% l('The catalog number you have entered looks like an Amazon ASIN.') %]
+        [% l('The {doc|status} describes how “official” the release is.',
+             { doc => { href => doc_link('Release#Status'), target => '_blank' } }) %]
       </p>
       <p>
-        [% l('If this is an Amazon ASIN, please do not enter it in the catalog number field, but add it as a URL relationship later.') %]
+        [% l('Please see the {doc|style guidelines} for more information.',
+             { doc => { href => doc_link('Style/Release#Status'), target => '_blank' } }) %]
       </p>
     <!-- /ko -->
   </div>
 
-  <div class="bubble" data-bind="bubble: $root.packagingBubble">
+  <div class="bubble" data-bind="bubble: $root.languageBubble">
+    <p>
+      [% l('The {doc|language} attribute should be set to the language used for the release title and track titles.
+            It should not be set to the language the lyrics are written in,
+            nor to the language used for other extra information on the cover.',
+           { doc => { href => doc_link('Release#Language'), target => '_blank' } }) %]
+    </p>
     <p>
-      [% l('Review the {packaging|list of packaging types} for help.',
-           { packaging => { href => doc_link('Release/Packaging'), target => '_blank' } }) %]
+      [% l('Please see the {doc|style guidelines} for more information.',
+           { doc => { href => doc_link('Style/Release#Language_and_script'), target => '_blank' } }) %]
     </p>
   </div>
 
-  <div class="bubble" data-bind="bubble: $root.languageBubble">
+  <div class="bubble" data-bind="bubble: $root.scriptBubble">
     <p>
-      [% l('The language attribute should be set to the language used for the release title and track titles. It should not be set to the language the lyrics are written in, nor to the language used for other extra information on the cover.') %]
```

Comment:
Maybe I'm not reading it properly, but are we removing the language bubble? There are a few other removals that I am not sure about.